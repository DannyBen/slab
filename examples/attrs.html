<div>
</div>
<div>
</div>
<div class="a">
</div>
<div class="a">
</div>
<div class="a">
</div>
<div class="a">
</div>
<div class="a b">
</div>
<div class="a b">
</div>
<div class="a b">
</div>
<div class="a b">
</div>
