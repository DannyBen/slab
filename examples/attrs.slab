div()

()

.a()

().a

div(class='a')

(class='a')

(class='a').b

.a(class='b')

(class='a', class='b')
