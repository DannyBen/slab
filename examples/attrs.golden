<div>
</div>
<div>
</div>
<div class="a">
</div>
<div class="a">
</div>
